.github/workflows/package.yaml

```diff
@@ -1,13 +1,18 @@
 name: Conda packaging and deployment
 
 on:
-  workflow_dispatch:
-  push:
-    branches: [qa, main]
-    tags: ['v*']
+  workflow_run:
+    workflows: [ "unit-test" ]  # must match the 'name:' of the unit test workflow
```

Comment:
```suggestion
    workflows: [ "unit-test" ]  # must match the 'name:' of the unit test workflow
    branches: [qa, main]
```

Reply:
The goal is to test the package build already on pull requests. The only difference in workflow will be the publish step, which only happens on tags.

However, the `workflow_run` directive is not as flexible as we had hoped. It seems to only run if the triggering workflow is run on the repo default branch. It was not triggered on this PR.